tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -351,7 +351,9 @@ TangoErrorType TangoRosNode::TangoSetupConfig() {
     return result;
   }
   const char* config_enable_depth = "config_enable_depth";
-  result = TangoConfig_setBool(tango_config_, config_enable_depth, true);
+  bool enable_depth;
+  node_handle_.param<bool>(ENABLE_DEPTH, enable_depth, true);
+  result = TangoConfig_setBool(tango_config_, config_enable_depth, enable_depth);
```

Comment:
Does this mean that if the parameter is changed between ROS startup and (re)connection to Tango, it would not be picked up? Is this what we want?